Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -481,7 +481,10 @@ void CGameListCtrl::InsertItemInReportView(long _Index)
 		}
 	}
 
-	SetItem(_Index, COLUMN_TITLE, StrToWxStr(name), -1);
+	if (rISOFile.IsDiscTwo() && !name.find("DISC TWO", 0))
```

Comment:
Does searching for "DISC TWO" really work with game names in languages other than English?